nf_core/lint.py

```diff
@@ -226,25 +226,27 @@ def check_files_exist(self):
         """
 
         # NB: Should all be files, not directories
-        # Supplying a list means if any are present it's a pass
+        # Supplying a list means if any are present it's a pass - lists
```

Comment:
Not sure that the comments is more clear with the extra ` - lists`

Reply:
The comment below says `- strings`. It's just a note to future me that this should be a list of lists, and the other a list of strings. It wasn't very important, but yes I agree it's not super clear.

Reply:
```suggestion
        # List of lists. Passes if any of the files in the sublist are found.
```

Reply:
I'm usually looking at Python without understanding much, and this was even less that usual, so I was concerned, but if you understand what you meant, it's good enough I think